doc/source/guidelines/version_support.rst

```diff
@@ -0,0 +1,38 @@
+Python Version Support
+======================
+
+When writing Python libraries, plan on supporting the oldest actively supported
+version of Python. For a quick reference, visit `Python EOL
+<https://endoflife.date/python>`_ summarized (as of 2022) here:
```

Comment:
This link is blocked for me, maybe we should point to https://devguide.python.org/#status-of-python-branches instead?

Reply:
Done. It's a bit more official as well.